docs/stcal/ramp_fitting/description.rst

```diff
@@ -284,3 +284,18 @@ For the optional output product, the variance of the slope due to the Poisson
 noise of the segment-specific slope is written to the VAR_POISSON extension.
 Similarly, the variance of the slope due to the read noise of the
 segment-specific slope  is written to the VAR_RNOISE extension.
+
+Data Quality Propagation
+=================
+
+If all groups in an integration are flagged as DO_NOT_USE or SATURATED, then
+the rateints product will be flagged as DO_NOT_USE.  For the rates product, the
+DO_NOT_USE flag will be set if all integrations are flagged as DO_NOT_USE.
+
+If all groups in an integration are flaggd as SATURATED, then the rateints
```

Comment:
Not obvious on first reading how this second case differs from first case. The wording above "If all groups in an integration are flagged as DO_NOT_USE or SATURATED" sounds like it already covers this subset case where all groups are flagged as SATURATED. In the first one, do you mean a mixture of either DO_NOT_USE or SATURATED, such that all groups are flagged with one or the other (e.g. groups 0 and 1 are DO_NOT_USE and all the rest are SATURATED)?

Reply:
I have updated this section to try to make it more clear.  Please let me know if what is there now is more understandable.